Price type is not always available

Some categories do not have a price type. Publishing failes then.

```
[ERROR] Traceback (most recent call last):
  File "/home/admin/Development/kleinanzeigen-bot/kleinanzeigen_bot/selenium_mixin.py", line 257, in web_await
    return WebDriverWait(self.webdriver, timeout).until(condition)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/admin/Development/kleinanzeigen-bot/.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py", line 95, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/home/admin/Development/kleinanzeigen-bot/kleinanzeigen_bot/__main__.py", line 9, in <module>
    kleinanzeigen_bot.main(sys.argv)
  File "/home/admin/Development/kleinanzeigen-bot/kleinanzeigen_bot/__init__.py", line 1034, in main
    KleinanzeigenBot().run(args)
  File "/home/admin/Development/kleinanzeigen-bot/kleinanzeigen_bot/__init__.py", line 82, in run
    self.publish_ads(ads)
  File "/home/admin/Development/kleinanzeigen-bot/kleinanzeigen_bot/__init__.py", line 462, in publish_ads
    self.publish_ad(ad_file, ad_cfg, ad_cfg_orig)
  File "/home/admin/Development/kleinanzeigen-bot/kleinanzeigen_bot/__init__.py", line 525, in publish_ad
    self.web_select(By.XPATH, "//select[@id='price-type-react' or @id='micro-frontend-price-type' or @id='priceType']", price_type)
  File "/home/admin/Development/kleinanzeigen-bot/kleinanzeigen_bot/selenium_mixin.py", line 393, in web_select
    elem = self.web_await(
           ^^^^^^^^^^^^^^^
  File "/home/admin/Development/kleinanzeigen-bot/kleinanzeigen_bot/selenium_mixin.py", line 260, in web_await
    raise exception_on_timeout() from ex
selenium.common.exceptions.NoSuchElementException: Message: Element xpath='//select[@id='price-type-react' or @id='micro-frontend-price-type' or @id='priceType']' not found or not clickable; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception


```

--- a/kleinanzeigen_bot/__init__.py
+++ b/kleinanzeigen_bot/__init__.py
@@ -522,7 +522,10 @@ class KleinanzeigenBot(SeleniumMixin):
         #############################
         price_type = ad_cfg["price_type"]
         if price_type != "NOT_APPLICABLE":
-            self.web_select(By.XPATH, "//select[@id='price-type-react' or @id='micro-frontend-price-type' or @id='priceType']", price_type)
+            try:
+                self.web_select(By.XPATH, "//select[@id='price-type-react' or @id='micro-frontend-price-type' or @id='priceType']", price_type)
+            except NoSuchElementException:
+                pass
             if safe_get(ad_cfg, "price"):
                 self.web_input(By.XPATH, "//input[@id='post-ad-frontend-price' or @id='micro-frontend-price' or @id='pstad-price']", ad_cfg["price"])
 

